extension/src/popup/components/sendPayment/SendAmount/AssetSelect/index.tsx

```diff
@@ -26,7 +26,7 @@ export function AssetSelect({
 }) {
   const dispatch = useDispatch();
   const { assetIcons } = useSelector(transactionSubmissionSelector);
-  const isScamAsset = useIsScamAsset(assetCode, issuerKey);
+  const isScamAsset = useIsOwnedScamAsset(assetCode, issuerKey);
```

Comment:
same change here - let's change `isScamAsset` to `isOwnedScamAsset` so a dev will understand what is when they see it being used below